htdocs/install/mysql/migration/19.0.0-20.0.0.sql

```diff
@@ -290,3 +290,6 @@ ALTER TABLE llx_receptiondet_batch CHANGE COLUMN fk_commande fk_element integer;
 ALTER TABLE llx_receptiondet_batch CHANGE COLUMN fk_commandefourndet fk_elementdet integer;
 
 ALTER TABLE llx_supplier_proposaldet MODIFY ref_fourn VARCHAR(128) NULL;
+
+ALTER TABLE llx_rights_def ADD COLUMN module_origin varchar(64) AFTER module;
```

Comment:
Difference between module and module_origin is not clear.
One is the name of module the permission is for
The other is the module that provide the permission. But which one is which one ?